octavia-cli/integration_tests/configurations/connections/poke_to_pg_secured/configuration.yaml

```diff
@@ -0,0 +1,361 @@
+# Configuration for connection poke_to_pg
```

Comment:
Do you need to create this file? Why don't you use `poke_to_pg/configuration.yaml`?